src/pages/management/changelog/index.mdx

```diff
@@ -28,6 +28,9 @@ The developer preview version provides a preview of the upcoming changes to the
 ### Webhooks
 - The [**Configuration**](/management/webhooks/v3.6#configuration) webhooks now contain a new `requester` property. The property contains details of who triggered the webhook: `user_id`, `client_id` and `account_id`.
 
+### Other
+- There is a new method `reactivate_email` that reactivates bounced email addresses.
```

Comment:
Let's do it this way (just please link the correct link as I linked the link from the deploy-preview😜) - There is a new [Reactivate email](https://deploy-preview-1332--livechat-public-docs.netlify.app/management/configuration-api/v3.6#reactivate-email) method that reactivates bounced email addresses.